packages/theme-dark-extension/style/variables.css

```diff
@@ -369,6 +369,20 @@ all of MD as it is not optimized for dense, information rich UIs.
   --jp-mirror-editor-error-color: #f00;
   --jp-mirror-editor-hr-color: #999;
 
+  /*
+    RTC user specific colors.
+    These colors are used for the cursor, username in the editor,
+    and the icon of the user.
+  */
+
+  --jp-collaborator-1: #ad4a00;
```

Comment:
perhaps `--jp-collaborator-color1` for parity with other naming schemes? presumably we could add some other distinguisher (e.g. border) that would give us 7 x 7 unique combinations or something.

Reply:
That's a great idea. Thanks, @bollwyvl.
I'll apply the changes!